examples/hyperapp-kitchen-sink/.babelrc

```diff
@@ -0,0 +1,11 @@
+{
+  "presets": ["env"],
+  "plugins": [
+    [
+      "transform-react-jsx",
+      {
+        "pragma": "h"
+      }
+    ]
+  ]
```

Comment:
How do you feel about adding it automatically like `@storybook/react` [does](https://github.com/storybooks/storybook/blob/master/app/react/src/server/wrapDefaultBabelConfig.js)?